### PR TITLE
feat(frontend): cross-chain campaigns, i18n and minor improvements

### PR DESCRIPTION
**Pull request by luzzif**

### CODE REVIEW

File: packages/frontend/src/components/campaigns/campaign/chain/index.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+"use client";
+
+import type { SupportedChain } from "@metrom-xyz/contracts";
+import styles from "./styles.module.css";
```
Comment (guerrap) [inline packages/frontend/src/components/campaigns/campaign/chain/index.tsx:4]: Styles as last import.

File: packages/frontend/src/components/campaigns/index.tsx [hunk 2]
```diff
@@ -2,35 +2,36 @@
...
 import { Campaign } from "./campaign";
 
 import styles from "./styles.module.css";
-
-const PAGE_SIZE = 10;
+import { useTranslations } from "next-intl";
 
 export function Campaigns() {
-    const [pageNumber, setPageNumber] = useState(1);
+    const t = useTranslations();
```
Comment (guerrap) [inline packages/frontend/src/components/campaigns/index.tsx:12]: We can add `allCampaigns` as a param to this hook to have all the local `t` scoped under that key.

File: packages/frontend/src/components/campaigns/campaign/chain/index.tsx (hunk 1 above)
Comment (luzzif) [reply packages/frontend/src/components/campaigns/campaign/chain/index.tsx]: Oh right, addressed in 29e717d56185787009a3d86f3a1966ed97f8c112

File: packages/frontend/src/components/campaigns/index.tsx (hunk 2 above)
Comment (luzzif) [reply packages/frontend/src/components/campaigns/index.tsx]: Right, addressed in 29e717d56185787009a3d86f3a1966ed97f8c112

**Review by guerrap — APPROVED** [review]

**Pull request merged**